### ISSUE TITLE
Test Failure: com.ibm.ws.logging.fat.TimeBasedLogRolloverTest.testInvalidRolloverInterval

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#21052]
RTC 290559

```
testInvalidRolloverInterval:java.lang.RuntimeException: 2022-05-04-02:29:06:077 Timed out waiting for [] and/or searching for [CWWKG001[7-8]I] in log file: C:/Users/jazz_build/Build/jazz-build-engines/wasrtc/EBCPROD/build/dev/image/output/wlp/usr/servers/com.ibm.ws.logging.timedrolloverxml/logs/messages.log. Extra info: value of (watchFor.size() > 0): true, value of (notStartedAppNames.size() > 0): false
at componenttest.topology.impl.LibertyServer.waitForConfigUpdateInLogUsingMark(LibertyServer.java:5505)
at componenttest.topology.impl.LibertyServer.waitForConfigUpdateInLogUsingMark(LibertyServer.java:5366)
at com.ibm.ws.logging.fat.TimeBasedLogRolloverTest.setServerConfiguration(TimeBasedLogRolloverTest.java:465)
at com.ibm.ws.logging.fat.TimeBasedLogRolloverTest.testInvalidRolloverInterval(TimeBasedLogRolloverTest.java:354)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:199)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:320)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:173)

```

**Comment by fmhwong** [comment]
@ellen-lau There are new occurrences associated with the RTC defect.  Can you take a look?

**Comment by ellen-lau** [comment]
fixed by: #21733
